extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -80,13 +80,20 @@ export class AmpSidebar extends AMP.BaseElement {
       this.element.setAttribute('aria-hidden', 'true');
     }
 
+    //Prepare for animation (will not animate the initial state).
+    this.mutateElement(() => {
+      this.element.classList.add('-amp-sidebar-animate');
+      setStyles(this.element, {
```

Comment:
Display is set here way too early. There's no guarantee that this menu will ever be open at this point. It should stay hidden until it is.